src/Symfony/Component/Form/Resources/translations/sf_form.en.xlf

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>No choice available</source>
+                <target><em>No choice available.</em></target>
```

Comment:
Either the <em> tag should be put in the correct xmlns or the whole content of <target> should be marked CDATA